scripts/india_census/common/utils.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def titleCase(st):
```

Comment:
should this be snake case?

Reply:
StatisticalVariable names are mixed. For example `Percent_Parent_25OrMoreYears_ChildEnrolledInPublicSchool_EducationalAttainment9ThTo12ThGradeNoDiploma`  has _ to separate but `EducationalAttainment9ThTo12ThGradeNoDiploma` is kind of title case. I am using this function to generate that name.

Reply:
Done.